.github/workflows/_update-inventory.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     name: Update Inventory
     runs-on: pub-hk-ubuntu-22.04-small
     steps:
-      - uses: heroku/use-app-token-action@main
+      - uses: actions/create-github-app-token@v1
         id: generate-token
         with:
           app_id: ${{ vars.LINGUIST_GH_APP_ID }}
```

Comment:
The options the new action takes use kebab-case rather than underscores:
https://github.com/actions/create-github-app-token?tab=readme-ov-file#inputs